### ISSUE TITLE
"feature/ethernet" merge broke examples

### ISSUE THREAD

**Issue by mabeett** [ciaa/Firmware#403]
After PR #401 blinking[0] example build is not working anymore.

```
./modules/drivers/cortexM4/lpc43xx/lpc4337/src/ciaaDriverEth.c:52:26: fatal error: arch/lpc_phy.h: No such file or directory
 #include "arch/lpc_phy.h"
                          ^
compilation terminated.
make: *** [ciaaDriverEth.o] Error 1
```

Adding `MODS ?= externals$(DS)lwip` to the project's makefile, the system the error is regardin `lwipopts.h` file:

```
./externals/lwip/inc/lwip/opt.h:45:22: fatal error: lwipopts.h: No such file or directory
 #include "lwipopts.h"
```

[0]: (and many others)

**Comment by mabeett** [comment]
WIP

**Comment by mabeett** [comment]
solved by #409
